lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -43,7 +43,7 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
 }
 
 .nx-collapsible-items--disabled {
-  .nx-collapsible-items__text, .nx-collapsible-items__child {
+  .nx-collapsible-items__text, .nx-collapsible-items__twisty {
```

Comment:
The ".nx-collapsible-items__trigger` referenced in the selector below contains both the child and the twisty, so it'd be better to just move the `color` down there

Reply:
This is fixed here e031b37beb3d0c2e0a02591f351e5987b0f42c22, thanks